mlflow/sklearn/utils.py

```diff
@@ -291,10 +291,18 @@ def _get_classifier_artifacts(fitted_estimator, prefix, X, y_true, sample_weight
     def plot_confusion_matrix(*args, **kwargs):
         import matplotlib
 
-        num_classes = len(set(y_true))
+        if hasattr(fitted_estimator, "classes_"):
+            classes = fitted_estimator.classes_
+        elif hasattr(fitted_estimator, "estimator") and hasattr(
+            fitted_estimator.estimator, "classes_"
```

Comment:
This is for cross validator case ?

Reply:
yep

Reply:
shall we handle special case for pipeline ?

Reply:
nit: we can put the logic in a helper function. Other code might reuse it in future.